### PR TITLE
Update doc for syslog possible values (EV-2821,EV-2822)

### PR DESCRIPTION
**Pull request by vara2504**
## Description

Add 2 doc change for syslog

1) Added possible values for EncryptionOption in command to generated in docs.
2) Updated the commands for existing logtypes since it was misleading from the actual behaviour.

Actual behavior:
When empty list is set, it will prepopulated with DNS,Audit and flow logs.
More details in the https://github.com/tigera/operator/blob/master/pkg/controller/logcollector/logcollector_controller.go#L213

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

File: api/v1/logcollector_types.go [hunk 1]
```diff
@@ -45,6 +45,9 @@ const (
 	CollectProcessPathDisable CollectProcessPathOption = "Disabled"
 )
 
+// EncryptionOption specifies traffic encryption mode to the Syslog server.
```
Comment (rene-dekker) [inline api/v1/logcollector_types.go:48]: // EncryptionOption specifies the traffic encryption mode when connecting to a Syslog server

File: api/v1/logcollector_types.go [hunk 2]
```diff
@@ -132,8 +135,8 @@ type SyslogStoreSpec struct {
 	// +optional
 	PacketSize *int32 `json:"packetSize,omitempty"`
 
-	// LogTypes contains a list of types of logs to export to syslog. By default, if this field is
-	// omitted, it will be set to include all possible values.
+	// LogTypes contains a list of types of logs to export to syslog. If empty list is set,
```
Comment (rene-dekker) [inline api/v1/logcollector_types.go:138]: Maybe this is better: 
> If no values are provided, the list will be updated to include log types Audit, DNS and Flows.
> Default: Audit, DNS, Flows

File: api/v1/logcollector_types.go (hunk 1 above)
Comment (vara2504) [reply api/v1/logcollector_types.go]: Updated

File: api/v1/logcollector_types.go (hunk 2 above)
Comment (vara2504) [reply api/v1/logcollector_types.go]: Done

**Comment by vara2504** [comment]
@rene-dekker Addressed your comments. Please take a look

**Review by rene-dekker — APPROVED** [review]
lgtm

**Pull request merged**